frontend/src/components/patient/PatientHistory.js

```diff
@@ -15,6 +16,11 @@ const PatientHistory = () => {
     }
   };
 
+  const [newPatientTab, setNewPatientTab] = useState({
```

Comment:
@tendomart , i dont see any palce exactly where this state is updated.
Is this still WIP ??

Reply:
that was removed